cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs

```diff
@@ -363,6 +359,11 @@ runTxBuildRaw (AnyCardanoEra era)
     firstExceptT ShelleyTxCmdWriteFileError . newExceptT $
       writeFileTextEnvelope fpath Nothing txBody
 
+joinEither :: (x -> z) -> (y -> z) -> Either x (Either y a) -> Either z a
```

Comment:
There are identical definitions of these functions. Let put them in `Cardano.CLI.Helpers`